packages/paste-libraries/react-textarea-autosize/src/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const noop = (): void => {};
+
+export const pick = <Obj extends {[key: string]: any}, Key extends keyof Obj>(props: Key[], obj: Obj): Pick<Obj, Key> =>
+  props.reduce((acc, prop) => {
+    acc[prop] = obj[prop];
+    return acc;
+  }, {} as Pick<Obj, Key>);
```

Comment:
question: does lodash pick do the same things? https://lodash.com/docs/4.17.15#pick? Should we use it instead?